Packages/com.unity.inputsystem/Documentation~/ActionsEditor.md

```diff
@@ -0,0 +1,134 @@
+
+![Composite Part](./Images/CompositePart.png)
+
+You can assign multiple Bindings to the same part. You can also duplicate individual part Bindings: right-click the Binding, then select **Duplicate** to create new part Bindings for the Composite. This can be used, for example, to create a single Composite for both "WASD" style controls and arrow keys.
+
+![Duplicated Part Bindings](./Images/DuplicatedPartBindings.png)
+
+### Editing Control Schemes
+
+Input Action Assets can have multiple [Control Schemes](ActionBindings.md#control-schemes), which let you enable or disable different sets of Bindings for your Actions for different types of Devices.
+
+![Control Scheme Properties](Images/ControlSchemeProperties.png)
+
+To see the Control Schemes in the Input Action Asset editor window, open the Control Scheme drop-down list in the top left of the window. This menu lets you add or remove Control Schemes to your Asset. If the Asset contains any Control Schemes, you can select a Control Scheme, and then the window only shows bindings that belong to that Scheme. If you select a binding, you can now pick the Control Schemes for which this binding should be active in the __Properties__ view to the left of the window. When you add a new Control Scheme, or select an existing Control Scheme, and then select __Edit Control Scheme…__, you can edit the name of the Control Scheme and which devices the Scheme should be active for.
+
+## Using Input Action Assets
+
+### The Automaticaly-generated InputActions API
+
+The most convenient ways to work with your Input Actions in scripts is to use the automatically generated C# API in the InputActions class.
+
+Each Action Map and Action you define in the Input Actions editor has its own corresponding C# member on the InputActions class.
+
+For example, if you have an Action Map called *"Player"* which defines *"Move"* as an Action with a 2D axis, and *"Jump"* as an button-type action, you can use a single line of code to read the values from each of these actions:
+
+```
+    // read the default "move" action, which is a 2D vector axis
+    Vector2 move = InputActions.player.move.value;
+
+    // read the default "jump" action, which is a true/false button
+    bool jump = InputActions.player.jump.isPressed;
+```
+
+**Note**: The Action Map and Actions in this example configured as defaults when you install the Input System package.
+
+### Using Actions with the `PlayerInput` component
+
+The [Player Input](PlayerInput.md) component provides a convenient way to handle input for one or multiple players. It requires you to set up all your Actions in the Input Actions editor, which you can then select in the Player Input component. The Player Input component can then automatically handle activating Action Maps and selecting Control Schemes for you.
+
+![PlayerInput](Images/PlayerInput.png)
+
+### Modifying Input Action Assets at runtime
```

Comment:
This should also be true for Project-wide Actions right? Not sure users will realize its treated as an asset